python/fedml/computing/scheduler/scheduler_entry/cluster_manager.py

```diff
@@ -0,0 +1,168 @@
+                response = requests.post(
+                    cluster_stop_url, verify=True, headers=cluster_api_headers, json=cluster_list_json
+                )
+            except requests.exceptions.SSLError as err:
+                MLOpsConfigs.install_root_ca_file()
+                response = requests.post(
+                    cluster_stop_url, verify=True, headers=cluster_api_headers, json=cluster_list_json
+                )
+        else:
+            response = requests.post(
+                cluster_stop_url, verify=True, headers=cluster_api_headers, json=cluster_list_json
+            )
+        if response.status_code != 200:
+            print(f"Stop cluster with response.status_code = {response.status_code}, "
+                  f"response.content: {response.content}")
+            return False
+        else:
+            resp_data = response.json()
+            code = resp_data.get("code", None)
+            data = resp_data.get("data", None)
+            if code is None or data is None or code == "FAILURE":
+                print(f"Stop cluster with response.status_code = {response.status_code}, "
+                      f"response.content: {response.content}")
+                return False
+
+        return True
+
+    def list_clusters(self, cluster_names=()):
+        cluster_list_result = None
+        cluster_list_url = ServerConstants.get_cluster_list_url(self.config_version)
+        cluster_api_headers = ServerConstants.API_HEADERS
+        cluster_list_json = {'cluster_names': list(set(cluster_names)), ClusterConstants.API_KEY: get_api_key()}
+        args = {"config_version": self.config_version}
+        _, cert_path = MLOpsConfigs.get_instance(args).get_request_params_with_version(self.config_version)
+        if cert_path is not None:
+            try:
+                requests.session().verify = cert_path
+                response = requests.post(
+                    cluster_list_url, verify=True, headers=cluster_api_headers, json=cluster_list_json
+                )
+            except requests.exceptions.SSLError as err:
+                MLOpsConfigs.install_root_ca_file()
+                response = requests.post(
+                    cluster_list_url, verify=True, headers=cluster_api_headers, json=cluster_list_json
+                )
+        else:
+            response = requests.post(
+                cluster_list_url, verify=True, headers=cluster_api_headers, json=cluster_list_json
+            )
+        if response.status_code != 200:
+            print(
+                f"Cluster list with response.status_code = {response.status_code}, response.content: {response.content}")
+            pass
+        else:
+            resp_data = response.json()
+            code = resp_data.get("code", None)
+            data = resp_data.get("data", None)
+            if code is None or data is None or code == "FAILURE":
+                print(f"List job with response.status_code = {response.status_code}, "
```

Comment:
The text 'List job' should be 'Cluster list'.

Reply:
Good catch. Will fix. Thanks